packages/core/src/components/popover-core/tds-popover-core.scss

```diff
@@ -0,0 +1,17 @@
+:host {
+  visibility: hidden;
+  opacity: 0;
+  pointer-events: none;
+}
+
+/* Fade animation rules when animation is 'fade' */
+:host(.tds-popover-animation-fade) {
+  transition: opacity 0.3s ease-in-out, visibility 0.3s ease-in-out;
```

Comment:
I'm guessing some CSS variables should be used here?

Reply:
Yes, probably some from the [_motion.scss](https://github.com/scania-digital-design-system/tegel/pull/863/files#diff-799504396d48367c4d9b82f4b61825a7ad962f083c5b6df941fd03d3991e01c7R1)